src/features/animation/video-exporter.tsx

```
@@ -0,0 +1,375 @@
+          width,
+          height,
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          justifyContent: 'center',
+          position: 'relative',
+          letterSpacing: 0,
+        }}
+      >
+        {/* Aggressive letter-spacing reset for video export */}
+        <style dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)